### DISCUSSION TITLE
Discord Ban

### DISCUSSION THREAD

**Discussion by Bogdanbarnus** [chromiecraft/chromiecraft#5541] [General]
Dear ChromieCraft Staff!

I want to introduce myself with my in-game name: Volpe!
I'm playing on the server since April/21 and i'm a big fan of it at the same time!
What i want to say is that once upon a time, my discord account got " scammed " or idk to be honest, if I remember right i've opened up a virus link.  After that I didn't have access to my account for like 1.5 week or so. I ended up with a ban from the Chromie Discord server, because of that issue. I want to appeal for an unban, because i feel like I belong to the community!
Thanks for your time! :)

**Comment by Seperest** [comment]
I feel your pain. Joined the server 2 months ago, clicked the invite link on the day I joined, already banned. My guess is whoever lived in my apt before me got IP banned? Been appealing for those 2 months and never got a word back. Sucks not being able to report bugs/bots.
